distribution/dashboard/ui/src/pages/Tenant/Detail/Backup/index.tsx

```diff
@@ -0,0 +1,67 @@
+import EmptyImg from '@/assets/empty.svg';
+import { getNSName } from '@/pages/Cluster/Detail/Overview/helper';
+import { getBackupPolicy, getTenant } from '@/services/tenant';
+import { PageContainer } from '@ant-design/pro-components';
+import { history } from '@umijs/max';
+import { useRequest } from 'ahooks';
+import { Button, Card, Row } from 'antd';
+import BasicInfo from '../Overview/BasicInfo';
+import BackupConfiguration from './BackupConfiguration';
+import BackupJobs from './BackupJobs';
+
+export default function Backup() {
+  const [ns, name] = getNSName();
+
+  const { data: backupPolicyResponse } = useRequest(getBackupPolicy, {
```

Comment:
const { data: backupPolicy } = useRequest(async () => {const res = await getBackupPolicy({ ns, name }) return res?.data});

Reply:
I plan to implement a useRequest encapsulation layer, so there's no need to address it individually here.